Release 1.1.0 (#1952)
prometheus/node_exporter: CHANGELOG.md

## master / unreleased

* [CHANGE]
* [FEATURE]
* [ENHANCEMENT]
* [BUGFIX]

## 1.1.0 / 2021-02-05

> * [CHANGE] Improve filter flag names. #1743

Reviewer: ```suggestion
* [CHANGE] Improve filter flag names #1743
```

Reviewer: some details maybe , or a note that it is backwards compat?

* [CHANGE] Add btrfs and powersupplyclass to list of exporters enabled by default #1897

> * [FEATURE] Add fibre channel collector #1786

Reviewer: Can you add that to the README as well?

Author: This is already in the README.

> * [FEATURE] Expose cpu bugs and flags as info metrics. #1788

Reviewer: Can you group the collectors together?

Author: I'm not sure what you mean. They are order by PR #.

Reviewer: Ok :) It was strange to see that in the middle of the new collectors.

> * [FEATURE] Add network_route collector #1811

Reviewer: Can you add that to the README as well?

Author: Yes, I forgot to add that.

> * [FEATURE] Add zoneinfo collector #1922

Reviewer: Can you add that to the README as well?

* [ENHANCEMENT] Add more InfiniBand counters #1694
* [ENHANCEMENT] Add flag to aggr ipvs metrics to avoid high cardinality metrics #1709
* [ENHANCEMENT] Adding backlog/current queue length to qdisc collector #1732
* [ENHANCEMENT] Include TCP OutRsts in netstat metrics #1733
* [ENHANCEMENT] Add pool size to entropy collector #1753
* [ENHANCEMENT] Remove CGO dependencies for OpenBSD amd64 #1774
* [ENHANCEMENT] bcache: add writeback_rate_debug stats #1658
* [ENHANCEMENT] Add check state for mdadm arrays via node_md_state metric #1810
* [ENHANCEMENT] Expose XFS inode statistics #1870
* [ENHANCEMENT] Expose zfs zpool state #1878
* [ENHANCEMENT] Added an ability to pass collector.supervisord.url via SUPERVISORD_URL environment variable #1947
* [BUGFIX] filesystem_freebsd: Fix label values #1728
* [BUGFIX] Fix various procfs parsing errors #1735
* [BUGFIX] Handle no data from powersupplyclass #1747

> * [BUGFIX] udp_queues_linux.go: s/upd/udp/ in two error strings #1769

Reviewer: ```suggestion
* [BUGFIX] udp_queues_linux.go: change upd to udp in two error strings #1769
```

* [BUGFIX] Fix node_scrape_collector_success behaviour #1816
* [BUGFIX] Fix NodeRAIDDegraded to not use a string rule expressions #1827

> * [BUGFIX] fix: node_md_disks state label from fail to failed #1862

Reviewer: ```suggestion
* [BUGFIX] Fix node_md_disks state label from fail to failed #1862
```

* [BUGFIX] Handle EPERM for syscall in timex collector #1938

> * [BUGFIX] bcache: fix typo #1943

Reviewer: ```suggestion
* [BUGFIX] bcache: fix typo in a metric name #1943
```
